src/Components/WebAssembly/WebAssembly/src/Hosting/RootComponentMappingCollection.cs

```diff
@@ -48,6 +48,27 @@ public void Add(Type componentType, string selector)
             Add(new RootComponentMapping(componentType, selector));
         }
 
+        /// <summary>
+        /// Adds a component mapping to the collection.
+        /// </summary>
+        /// <param name="componentType">The component type. Must implement <see cref="IComponent"/>.</param>
+        /// <param name="selector">The DOM element selector.</param>
+        /// <param name="parameters">The parameters to the root component.</param>
+        public void Add(Type componentType, string selector, ParameterView parameters)
```

Comment:
Could you change the 2-param overload above to call the 3-param one, passing `ParameterView.Empty`?

Reply:
done